src/test/java/uk/gov/register/resources/v1/EntryResourceTest.java

```diff
@@ -17,15 +17,6 @@
 
 @RunWith(MockitoJUnitRunner.class)
 public class EntryResourceTest {
-
-    @Test
-    public void findByEntryNumberHtmlSupportsHtml() throws Exception {
```

Comment:
I was not aware we removed this functionality for V1, so should the test still work?

Reply:
I must have removed this by accident, I'll add it back